themes/default/content/product/_index.md

```diff
@@ -12,99 +12,203 @@ subheading: |
     and tame cloud complexity -- something we call Cloud Engineering.
 
 overview:
-    title: Unite developers, infrastructure teams, and security engineers
+    title: Infrastructure as Code for Engineers
```

Comment:
there's a lot of whitespace above the "Infrastructure as code for Engineers" and below the "Get Started" button. do you think it would look better reduced?

Reply:
agreed. I reduced it a bit.